gomint-server/src/main/java/io/gomint/server/inventory/item/ItemAcaciaLeaves.java

```diff
@@ -7,7 +7,6 @@
  * @author geNAZt
  * @version 1.0
  */
-@RegisterInfo( sId = "minecraft:leaves2")
```

Comment:
Why has this register been removed but not the implementation class? There is currently no way to use this and the API class on it